### ISSUE TITLE
Error: Image is not available

### ISSUE THREAD

**Issue by peduarte** [ensdomains/ens-avatar#21]
Hey there, I've been having issues loading the avatar for `nicolo.eth`.  Here's a link to a minimal reproducible example: https://codesandbox.io/s/competent-colden-fuq4xy?file=/src/App.js

It works with all other ENS I've tried, but for some reason, it crashed with `nicolo.eth`

The Avatar Record on ENS is: `eip155:1/erc721:0xabefbc9fd2f806065b4f3c237d4b59d9a97bcac7/25042` (apparently a Zora NFT)

Steps to reproduce:
- Open the MRE https://codesandbox.io/s/competent-colden-fuq4xy?file=/src/App.js

Related: https://github.com/raycast/extensions/issues/2816

Thanks!

**Comment by peduarte** [comment]
Update: also happens on `sammdec.eth` - which also has the Avatar Record set to The Zora Contract (`0xabefbc9fd2f806065b4f3c237d4b59d9a97bcac7`)

**Comment by serenae-fansubs** [comment]
That's an issue with that NFT contract. The creators/devs behind that NFT project have not implemented ERC-721 correctly.

https://eips.ethereum.org/EIPS/eip-721#specification

The tokenURI method should point to a JSON file following the schema outlined in the spec, but their contract does not do this.

**Comment by peduarte** [comment]
@serenae-fansubs makes sense! after a few hours of debugging with @sammdec we realised that too!

is it worth trying to catch this error from the library to prevent crashes?

on my current project, i'm basically ignoring checking for avatar if it uses the Zora Contract:
<img width="739" alt="CleanShot 2022-09-06 at 19 47 26@2x" src="https://user-images.githubusercontent.com/372831/188704407-7eb3ca11-249c-442a-b747-0411ca8b77a4.png">



Just an idea!
